webpage/_data/navigation.yml

```diff
@@ -9,11 +9,17 @@ main:
 site_nav:
   - title: Video Tutorials
     children:
+      - title: "NANOOS Visualization System (NVS)"
+        url: https://www.youtube.com/watch?v=MEVz0jOsqmI
       - title: "IOOS conda environment"
         url: https://github.com/ioos/notebooks_demos/wiki/Installing-Conda-Python-with-the-IOOS-environment
-      - title: "OWSLib"
-        url: https://geopython.github.io/OWSLib/
-      - title: "PyoosS"
-        url: https://github.com/ioos/pyoos
-      - title: "PyCSW"
-        url: http://pycsw.org/
+      - title: "Introduction to Data Explorer"
```

Comment:
We should indicate that these are tutorials for the "NANOOS Data Explorer". It should also be grouped with the first tutorial titled the "NANOOS Visualization System (NVS)"